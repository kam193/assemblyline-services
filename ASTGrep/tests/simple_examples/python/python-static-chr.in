x = chr(73) + (
        chr(73)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(73)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(73)
        + chr(73)
        + chr(73)
        + chr(73)
        + chr(108)
        + chr(108)
        + chr(108)
        + chr(73)
        + chr(73)
        + chr(108)
        + chr(108)
    )

y = chr(109)
z = chr(108) + chr(108) + chr(108) + chr(108)

chr(a)
