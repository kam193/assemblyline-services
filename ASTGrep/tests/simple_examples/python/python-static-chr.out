x = 'IIllllllllIlllIIIIlllIIll'

y = 'm'
z = 'llll'

chr(a)

